packages/excalidraw/utils.ts

```diff
@@ -984,6 +985,32 @@ export const memoize = <T extends Record<string, any>, R extends any>(
   return ret as typeof func & { clear: () => void };
 };
 
+export const memo = <P extends Array<unknown>, R>(
```

Comment:
if this is just a test util, let's move it to `packages/excalidraw/tests/test-utils.ts` and move the `deep-eql` to dev dependencies.

Note we already have `memoize` util just above this one, though it's shallow-only. But if we ended up using the deep comparison in normal code, we'd want to merge these instead of keeping the two.

Reply:
EDIT: actually, you're not even using it in tests, just testing the function itself. So it's unused, or is my intellisense and search failing me?

Reply:
I thought I needed it but no longer required. Removed entirely.